java-checks/src/main/java/org/sonar/java/checks/ReturnInFinallyCheck.java

```diff
@@ -77,10 +81,32 @@ public void visitMethod(MethodTree tree) {
 
   @Override
   public void visitReturnStatement(ReturnStatementTree tree) {
+   reportIssue(tree.returnKeyword(), "return");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)